src/components/Icons.tsx

```diff
@@ -582,8 +611,8 @@ function frx(flagA: keyof typeof FlagIconMap, flagB: keyof typeof FlagIconMap) {
 export const OTCBadgeIcon = Wrapper(OTCBadge);
 const BasketIndexIcon = Wrapper(BasketIndexPlaceholder);
 
-function otc(flag: keyof typeof FlagIconMap, symbol: string) {
-    const FlagIcon = FlagIconMap[flag];
+function otc(flag: keyof typeof StockIndicesMap, symbol: string) {
+    const FlagIcon = StockIndicesMap[flag];
     ItemIconMap[symbol as keyof typeof ItemIconMap] = FlagIcon;
 }
```

Comment:
```suggestion
function otc(flag: keyof typeof StockIndicesMap, symbol: string) {
    const StockIndexIcon = StockIndicesMap[flag];
    ItemIconMap[symbol as keyof typeof ItemIconMap] = StockIndexIcon;
}
```
@hamza-deriv or you can add these icons directly to ItemIconMap like it's done for synthetics (vol100, etc) since Stock indices (otc) are no longer simple flag icons as before